kasa/experimental/modules/camera.py

```diff
@@ -0,0 +1,42 @@
+"""Implementation of device module."""
+
+from __future__ import annotations
+
+from ...feature import Feature
+from ..smartcameramodule import SmartCameraModule
+
+
+class Camera(SmartCameraModule):
+    """Implementation of device module."""
+
+    NAME = "Camera"
+    QUERY_GETTER_NAME = "getLensMaskConfig"
+    QUERY_MODULE_NAME = "lens_mask"
+    QUERY_SECTION_NAMES = "lens_mask_info"
+
+    def _initialize_features(self):
+        """Initialize features after the initial update."""
+        if self.data:
```

Comment:
This behaves differently to the other implementations which raise an exception if the data is not available / the `update()` has not been called. I'd say that we should make a decision which way we want to go, and keep it consistent across the implementations.